.cico/deploy.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -xe
+
+# Make sure we are in the right directory
+cd $(dirname $(readlink -f $0))/../
+source .cico/setup.sh
+
+setup
+
+deploy
+
+# Just deploy once!
+if [ "$TARGET" != "rhel" ]; then
+    deploy_devcluster build || true # Don't fail on deploying the cluster it's not critical yet
```

Comment:
You can remove that, since you don't provide an openshift-instal.bash script (and not a req for now)